app/views/corporate/index.html.erb

```diff
@@ -15,7 +15,7 @@
           </a>
         </div>
         <div class="l-promo-corporate__item">
-          <a class="promo__link" href="/en/corporate_categories/media-centre">
+          <a class="promo__link" href="/en/corporate/media-centre">
```

Comment:
@tomas-stefano I've done this, and also addressed a problem I missed the first time around (missing Welsh links!). I've updated the PR description since this changes the scope quite a lot.